state/txindex/kv/kv_bench_test.go

```diff
@@ -0,0 +1,70 @@
+package kv
+
+import (
+	"crypto/rand"
+	"fmt"
+	"io/ioutil"
+	"testing"
+
+	abci "github.com/tendermint/tendermint/abci/types"
+	cmn "github.com/tendermint/tendermint/libs/common"
+	"github.com/tendermint/tendermint/libs/pubsub/query"
+	"github.com/tendermint/tendermint/types"
+	dbm "github.com/tendermint/tm-db"
+)
+
+func BenchmarkTxSearch(b *testing.B) {
+	dbDir, err := ioutil.TempDir("", "benchmark_tx_search_test")
+	if err != nil {
+		b.Errorf("failed to create temporary directory: %s", err)
+	}
+
+	db, err := dbm.NewGoLevelDB("benchmark_tx_search_test", dbDir)
+	if err != nil {
+		b.Errorf("failed to create database: %s", err)
+	}
+
+	allowedTags := []string{"transfer.address", "transfer.amount"}
+	indexer := NewTxIndex(db, IndexTags(allowedTags))
+
+	for i := 0; i < 35000; i++ {
+		events := []abci.Event{
+			{
+				Type: "transfer",
+				Attributes: []cmn.KVPair{
+					{Key: []byte("address"), Value: []byte(fmt.Sprintf("address_%d", i%100))},
+					{Key: []byte("amount"), Value: []byte("50")},
+				},
+			},
+		}
+
+		txBz := make([]byte, 8)
+		_, _ = rand.Read(txBz)
```

Comment:
why do we need _, _?

Reply:
Just to please any IDE. We can remove it if so desired.

Reply:
Updated.